.github/workflows/ci.yml

```
@@ -1076,9 +1079,9 @@ jobs:
         with:
           token: ${{ secrets.CANARY_DOCKER_BUILD }}
           workflow: .github/workflows/deploy.yml
-          ref: 'refs/heads/main'
+          ref: ${{ needs.job_setup.outputs.is_main == 'true' && 'refs/heads/main' || 'refs/heads/6.x' }}
           repo: TryGhost/Ghost-Moya
-          inputs: '{"version":"canary","environment":"staging","version_extra":"${{ env.branch_name }}"}'
+          inputs: ${{ needs.job_setup.outputs.is_main == 'true' && '{"version":"canary","environment":"staging","version_extra":"' + env.branch_name + '"}' || '{"version":"six","environment":"staging","version_extra":"' + env.branch_name + '"}' }}
```

actionlint (1.7.7)

got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)